backend/src/db-utils.js

```diff
@@ -587,6 +587,115 @@ const queryAccountTransactionsCount = async (accountId) => {
+         WHERE account_id = :account_id
+         LIMIT 1`,
+        { account_id: accountId },
+      ],
+      { dataSource: DS_ANALYTICS_BACKEND }
+    );
+    if (!query || !query.out_transactions_count) {
+      return 0;
+    }
+    return parseInt(query.out_transactions_count);
+  }
+  async function queryTransactionsCountFromIndexerForLastDay(accountId) {
+    const query = await querySingleRow(
+      [
+        `SELECT
+         COUNT(transactions.transaction_hash) AS out_transactions_count
+         FROM transactions
+         WHERE signer_account_id = :account_id
+         AND transactions.block_timestamp >= (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '1 day')) AS bigint) * 1000 * 1000 * 1000)
+         LIMIT 1`,
+        { account_id: accountId },
+      ],
+      { dataSource: DS_INDEXER_BACKEND }
+    );
+    if (!query || !query.out_transactions_count) {
+      return 0;
+    }
+    return parseInt(query.out_transactions_count);
+  }
+
+  const [analyticsTxCount, indexerTxCount] = await Promise.all([
+    queryOutcomeTransactionsCountFromAnalytics(accountId),
+    queryTransactionsCountFromIndexerForLastDay(accountId),
+  ]);
+  return analyticsTxCount + indexerTxCount;
+};
+
+const queryAccountIncomeTransactionsCount = async (accountId) => {
+  const {
+    last_day_collected_timestamp: lastDayCollectedTimestamp,
+  } = await querySingleRow(
+    [
+      `SELECT (CAST(EXTRACT(EPOCH FROM collected_for_day) AS bigint) * 1000 * 1000 * 1000) AS last_day_collected_timestamp
+       FROM daily_ingoing_transactions_per_account_count
+       ORDER BY collected_for_day DESC
+       LIMIT 1`,
+    ],
+    { dataSource: DS_ANALYTICS_BACKEND }
+  );
+  async function queryIncomeTransactionsCountFromAnalytics(accountId) {
+    const query = await querySingleRow(
+      [
+        `SELECT
+         SUM(ingoing_transactions_count) as in_transactions_count
+         FROM daily_ingoing_transactions_per_account_count
+         WHERE account_id = :account_id
+         LIMIT 1`,
+        { account_id: accountId },
+      ],
+      { dataSource: DS_ANALYTICS_BACKEND }
+    );
+    if (!query || !query.in_transactions_count) {
+      return 0;
+    }
+    return parseInt(query.in_transactions_count);
+  }
+
+  async function queryIncomeTransactionsCountFromIndexerForLastDay(
+    accountId,
+    lastDayCollectedTimestamp
+  ) {
+    const query = await querySingleRow(
+      [
+        `SELECT COUNT(DISTINCT transactions.transaction_hash) AS in_transactions_count
+         FROM transactions
+         LEFT JOIN receipts ON receipts.originated_from_transaction_hash = transactions.transaction_hash
+            AND transactions.block_timestamp > :last_day_collected_timestamp
+         WHERE receipts.included_in_block_timestamp > :last_day_collected_timestamp
+            AND transactions.signer_account_id != :requested_account_id
+            AND receipts.receiver_account_id = :requested_account_id`,
```

Comment:
I had a discussion about this query with @telezhnaya. She build a great strategy to collect the data in such way:
1. query for the last collected date from analytics;
2. convert date to timestamp;
3. put timestamp to this select.

This steps are needed to avoid missing data at the midnight for whole prev day.
It works, and runs super fast! (~1s for `near` account). But when I compare this query with this one
```
SELECT COUNT(DISTINCT transactions.transaction_hash)
FROM transactions
LEFT JOIN receipts ON receipts.originated_from_transaction_hash = transactions.transaction_hash
    AND transactions.block_timestamp > (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '1 day')) AS bigint) * 1000 * 1000 * 1000)
WHERE receipts.included_in_block_timestamp > (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '1 day')) AS bigint) * 1000 * 1000 * 1000)
    AND transactions.signer_account_id != :requested_account_id
    AND receipts.receiver_account_id = :requested_account_id
``` 
then the resulting value will differ by about 50 transactions (tested on `near` account). @frol what do you thinks about it?